jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -212,6 +213,55 @@ AC_DEFUN([OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU],
   esac
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_JITSERVER],
+[
+  AC_MSG_CHECKING([for jitserver])
+  AC_ARG_ENABLE([jitserver], [AS_HELP_STRING([--enable-jitserver], [enable JITServer support @<:@disabled@:>@])])
+  OPENJ9_ENABLE_JITSERVER=false
+
+  if test "x$enable_jitserver" = xyes ; then
+    if test "x$OPENJDK_TARGET_OS" = xlinux ; then
+      AC_CHECK_PROG(PROTOC_INSTALLED,protoc,yes,no)
+      if test "x$PROTOC_INSTALLED" = xyes ; then
+
+        AC_MSG_CHECKING([for protobuf version])
+        if test "x$OPENJ9_CPU" = xx86-64 ; then
+          MIN_SUPPORTED_PROTOBUF_VERSION=3.5.1
+        else
+          MIN_SUPPORTED_PROTOBUF_VERSION=3.7.1
+        fi
+        PROTOBUF_VERSION=`protoc --version 2>&1 | $SED -e 's/libprotoc //'`
+        AC_MSG_RESULT([$PROTOBUF_VERSION])
+
+        AS_VERSION_COMPARE([$PROTOBUF_VERSION], [$MIN_SUPPORTED_PROTOBUF_VERSION],
+          [PROTOBUF_VERSION_SUPPORTED=-1],
+          [PROTOBUF_VERSION_SUPPORTED=0],
+          [PROTOBUF_VERSION_SUPPORTED=1])
+        if test "x$PROTOBUF_VERSION_SUPPORTED" != x-1 ; then
+          AC_MSG_RESULT([yes (explicitly enabled)])
+          OPENJ9_ENABLE_JITSERVER=true
+        else
+          AC_MSG_ERROR([Require protobuf >= ($MIN_SUPPORTED_PROTOBUF_VERSION) for ($OPENJ9_CPU)])
+        fi
+
+      else
+        AC_MSG_ERROR([protoc not found])
+      fi
+    else
+      AC_MSG_RESULT([no (default for $OPENJDK_TARGET_OS)])
```

Comment:
This isn't the right message if `--enable-jitserver` was specified: it should say unsupported or something similar.

Reply:
Inverting these `if` statements would put the conditions closer to *both* branches making them easier to understand.